### PR TITLE
[Hub Generated] Review request for Microsoft.Consumption to add version 2018-11-01-preview

### PR DESCRIPTION
**Pull request by sandeepkatumalla**
If you are a MSFT employee you can view your [work branch via this link](https://portal.azure-devex-tools.com/app/branch/sandeepkatumalla/azure-rest-api-specs/dev-consumption-Microsoft.Consumption-2018-11-01-preview).

### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

To the reviewer, all of these changes have previously been approved already, but not merged because we weren't ready yet.

#4484
#4418
#3977

Now are APIs are live and so we'd like to finalize public documentation and swagger for 2018-11-01-preview

### CODE REVIEW

**Comment by adxsdknet** [comment]
# Automation for azure-sdk-for-net 
A PR has been created for you: 
https://github.com/azure/azure-sdk-for-net/pull/5517 
.NET SDK Commits: 
https://github.com/adxsdknet/azure-sdk-for-net/pull/5517/commits/8745050ca86c34f583a8bba95de64d0fbeacb7ea

**Review by ms-premp — APPROVED** [review]
Looks good to me.

**Comment by sandeepkatumalla** [comment]
> If you are a MSFT employee you can view your [work branch via this link](https://portal.azure-devex-tools.com/app/branch/sandeepkatumalla/azure-rest-api-specs/dev-consumption-Microsoft.Consumption-2018-11-01-preview).
> 
> ### Contribution checklist:
> * [x]  I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
> * [x]  [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

To the reviewer, all of these changes have previously been approved already, but not merged because we weren't ready yet.

https://github.com/Azure/azure-rest-api-specs/pull/4484
https://github.com/Azure/azure-rest-api-specs/pull/4418
https://github.com/Azure/azure-rest-api-specs/pull/3977

**Review by anuchandy — APPROVED** [review]
Content of this PR is already reviewed in following PRs.
#4484
#4418
#3977

File: specification/consumption/resource-manager/Microsoft.Consumption/preview/2018-11-01-preview/consumption.json [hunk 1]
```diff
@@ -0,0 +1,1476 @@
...
+        }
+      }
+    },
+    "/providers/Microsoft.Billing/billingAccounts/{billingAccountId}/providers/Microsoft.Consumption/charges": {
+      "get": {
+        "tags": [
+          "Charges"
+        ],
+        "operationId": "ChargesByBillingAccount_List",
+        "description": "Lists the charges by billingAccountId for given start and end date. Start and end date are used to determine the billing period. For current month, the data will be provided from month to date. If there are no chages for a month then that month will show all zeroes.",
```
Comment (nschonni) [inline specification/consumption/resource-manager/Microsoft.Consumption/preview/2018-11-01-preview/consumption.json]: chages -> charges

File: specification/consumption/resource-manager/Microsoft.Consumption/preview/2018-11-01-preview/consumption.json [hunk 2]
```diff
@@ -0,0 +1,1476 @@
...
+        }
+      }
+    },
+    "/providers/Microsoft.Billing/billingAccounts/{billingAccountId}/billingProfiles/{billingProfileId}/providers/Microsoft.Consumption/charges": {
+      "get": {
+        "tags": [
+          "Charges"
+        ],
+        "operationId": "ChargesByBillingProfile_List",
+        "description": "Lists the charges by billing profile id for given start and end date. Start and end date are used to determine the billing period. For current month, the data will be provided from month to date. If there are no chages for a month then that month will show all zeroes.",
```
Comment (nschonni) [inline specification/consumption/resource-manager/Microsoft.Consumption/preview/2018-11-01-preview/consumption.json]: chages -> charges

File: specification/consumption/resource-manager/Microsoft.Consumption/preview/2018-11-01-preview/consumption.json [hunk 3]
```diff
@@ -0,0 +1,1476 @@
...
+        }
+      }
+    },
+    "/providers/Microsoft.Billing/billingAccounts/{billingAccountId}/invoiceSections/{invoiceSectionId}/providers/Microsoft.Consumption/charges": {
+      "get": {
+        "tags": [
+          "Charges"
+        ],
+        "operationId": "ChargesByInvoiceSection_List",
+        "description": "Lists the charges by invoice section id for given start and end date. Start and end date are used to determine the billing period. For current month, the data will be provided from month to date. If there are no chages for a month then that month will show all zeroes.",
```
Comment (nschonni) [inline specification/consumption/resource-manager/Microsoft.Consumption/preview/2018-11-01-preview/consumption.json]: chages -> charges

File: specification/consumption/resource-manager/Microsoft.Consumption/preview/2018-11-01-preview/consumption.json [hunk 4]
```diff
@@ -0,0 +1,1476 @@
...
+					"description": "The countryCode for Enrollment",
+					"type": "string",
+					"readOnly": true
+				},
+				"status": {
+					"description": "Enrollment status",
+					"type": "string",
+					"readOnly": true
+				},
+				"billingCylce": {
```
Comment (nschonni) [inline specification/consumption/resource-manager/Microsoft.Consumption/preview/2018-11-01-preview/consumption.json]: Cylce -> Cycle

**Pull request merged**